src/wolfio.c

```diff
@@ -1108,17 +1108,30 @@ int wolfIO_HttpProcessResponse(int sfd, const char** appStrList,
 
             switch (state) {
                 case phr_init:
-                    if (XSTRLEN(start) < 15) { /* 15 is the length of the two
-                                          constant strings we're about to
-                                          compare against. */
+                    if (XSTRLEN(start) < 15 && XSTRLEN(start) == 12) {
+                                          /* "HTTP/1.1 200 OK" is length 15
+                                           * "HTTP/1.1 200" is length 12 */
+                        checkShort = 1;
+                    } else if (XSTRLEN(start) < 15) {
                         WOLFSSL_MSG("wolfIO_HttpProcessResponse HTTP header too short.");
                         return -1;
                     }
                     if (XSTRNCASECMP(start, "HTTP/1", 6) == 0) {
                         start += 9;
-                        if (XSTRNCASECMP(start, "200 OK", 6) != 0) {
-                            WOLFSSL_MSG("wolfIO_HttpProcessResponse not OK");
-                            return -1;
+                        if (!checkShort) {
```

Comment:
Why not just check for "200" since the "OK" is not significant. You only care about the substring "200", you could use `XSTRNSTR(start, "200", XSTRLEN(start));`